### ISSUE TITLE
Date for generated markdown?

### ISSUE THREAD

**Issue by dionjwa** [sillsdev/docu-notion#48]
How do have a date for a given notion generated page/blog entry?

Maybe it could be a notion doc property? But what about overwrites? What then is the correct date?

Basically, I want my `docu-notion` blog entries to have the correct date, but I'm not sure the best way to do this.

**Comment by hatton** [comment]
Yeah, maybe the date should be akin to the current support for a `slug` property? I'm not clear about your concern for overwrites.

I apologize, I know nothing of blogs and don't have time at the moment to look into features beyond what our group needs. If you do get totally stuck, let me know and when I can, I'll look at it.

**Comment by dionjwa** [comment]
Yeah, no need to apologize for anything, this piece of code is super useful to me, and requests or questions are completely optional to even answer. Gratitude for a useful tool.

**Comment by dionjwa** [comment]
What I need is a plugin to know the page id of whatever the context is. Currently, I see no mechanism to get the pageId anywhere in any of the plugins. Am I missing something?
